content/learn/keyconcepts.adoc

```diff
@@ -0,0 +1,164 @@
+
+When joining a managed cluster to Red Hat Advanced Cluster Management (RHACM) or deploying a new cluster with RHACM, it must be assigned to at least one ClusterGroup. RHACM identifies the managed cluster's membership in a ClusterGroup and proceeds to set up the cluster, including installing the RHACM agent. Once the setup is complete, RHACM deploys GitOps and supplies it with information about the ClusterGroup. GitOps then retrieves the associated values file and proceeds to deploy the Operators, configurations, and charts accordingly.
+
+For more information, see link:https://validatedpatterns.io/learn/clustergroup-in-values-files/[ClusterGroup configuration in values files]. 
+
+[id="GitOps"]
+== GitOps 
+
+GitOps is a way to manage cloud-native systems that are powered by Kubernetes. It leverages a policy-as-code approach to define and manage every layer of the modern application stack from infrastructure, networking application code, and the GitOps pipeline itself.
+
+The key principle of GitOps are: 
+
+* *Declarative*: The methodology requires describing the desired state, achieved through raw manifests, helm charts, kustomize, or other forms of automation.
+* *Versioned and immutability*: Git ensures versioning and immutability, serving as the definitive source of truth. Version control and historical tracking offer insights into changes that impact the clusters.
+* *Pulled automatically*: The GitOps controller pulls the state automatically to prevent any errors introduced by humans, and it also allows the application an opportunity to heal itself.
+* *Continuously reconciled*: The GitOps controller has a reconciliation loop that by default runs every 3 minutes. When the reconciler identifies a diff between git and the cluster, it will reconcile the change onto the cluster during the next synchronization.
+
+GitOps within the validated pattern framework ensures that infrastructure and application configurations are managed declaratively, consistently, and securely. GitOps ensures consistency across our environments, platforms and applications.  
+
+For more information, see link:https://docs.openshift.com/gitops/1.12/understanding_openshift_gitops/what-is-gitops.html[GitOps].
+
+[id="namespaces"]
+== Namespaces
+
+Namespaces in a validated pattern are essential for organizing and managing resources within an OpenShift Container Platform cluster, ensuring security, consistency, and efficient resource allocation. Recommendations for defining namespaces include using consistent naming conventions, ensuring isolation and security through policies and RBAC, setting resource quotas, tailoring configurations to specific environments, and designing namespaces for modularity and reusability across patterns or applications.
+
+Operators generally create their own namespace. But often you may need another different one however you should check and see if there are expected namespaces with the product. 
```

Comment:
suggestion:  "Operators generally create their own namespace. But often you may need another different one however you should check and see if there are expected namespaces with the product."
to  
Operators generally create their own namespaces, but you may need to create additional ones. Check if there are expected namespaces with the product before creating new ones.